### PR TITLE
test

### PR DESCRIPTION
**Pull request by xorserer**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/xorserer/Playground/pull/19" target="_blank">ReviewNinja

File: Class.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Created by Nir Bruner.
+// Copyright 2015.
+
+//documentation
+@interface Class : NSObject
+
+// this property is ble ble
+@property (nonatomic) BOOL bleble;
+
+// This is a conflicting property.
```
Comment (xorserer) [inline Class.h]: comment `hello` _la la la_

Comment (xorserer) [inline Class.h]: !fix

File: Class.mm [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Created by Nir Bruner.
+// Copyright 2016.
+
+#include "Class.h"
+
+@implementation Class
+
+- (instancetype)init {
```
Comment (xorserer) [inline Class.mm]: another issue here !fix

File: Class.h [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Created by Nir Bruner.
+// Copyright 2015.
+
+//documentation
```
Comment (xorserer) [inline Class.h]: non issue comment

Comment (xorserer) [inline Class.h]: !resolved

File: Class.h (hunk 1 above)
Comment (xorserer) [inline Class.h]: !resolved

File: Class.mm (hunk 2 above)
Comment (xorserer) [inline Class.mm]: !resolved

File: Class.mm [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// Created by Nir Bruner.
+// Copyright 2016.
+
+#include "Class.h"
+
```
Comment (xorserer) [inline Class.mm]: !fixed

**Pull request closed without merging**